src/main/kotlin/burp/BurpExtender.kt

```diff
@@ -578,13 +579,25 @@ class BurpExtender : IBurpExtender, ITab, ListDataListener, IHttpListener {
 
     private fun loadConfig(): Piper.Config {
         val serialized = callbacks.loadExtensionSetting(EXTENSION_SETTINGS_KEY)
```

Comment:
Why load it if you potentially will never use it anyway? I'd only load it if `env` is `null`, which would also make the code easier to read/follow.

Reply:
Good catch! Resolved in [2754328](https://github.com/silentsignal/burp-piper/pull/29/commits/2754328ec2f75e553fd397544fd1095e1f840f7d)